### ISSUE TITLE
distal radioulnar joint (DRUJ) arthritis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10719]
**Preferred term label:**
distal radioulnar joint (DRUJ) arthritis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Widening of the DRUJ interval, volar/dorsal displacement of the distal ulna, radiographic evidence of osteoarthritis at the DRUJ joint including osteophytes, subchondral cysts, subchrondal sclerosis, joint space narrowing

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Distal radioulnar joint (DRUJ) arthritis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001130
- label: Distal radioulnar joint arthritis
- parents: [Abnormality of the radioulnar joints (HP:0003059)](https://hpo.jax.org/app/browse/term/HP:0003059)
- pmid: PMID:23027692
- def: An inflammatory condition of the joint that connects the far ends of the radius and the ulna, known as the distal radioulnar joint (DRUJ). DRUJ arthritis may be characterized by widening of the DRUJ interval, volar/dorsal displacement of the distal ulna, radiographic evidence of osteoarthritis at the DRUJ joint including osteophytes, subchondral cysts, subchrondal sclerosis, and joint space narrowing.
- comment: Arthritis of the distal radioulnar joint (DRUJ) between the sigmoid notch of the distal radius and the ulnar head can be caused by traumatic, inflammatory, congenital, and degenerative processes.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10719
- synonyms: DRUJ arthritis(NONE)|Arthritis of the distal radioulnar joint(NONE)
